app/src/main/res/values/strings.xml

```diff
@@ -152,7 +152,7 @@
 
     <!-- Reminder notifications -->
     <string name="reminder_notification_title">Do you have any symptoms?</string>
-    <string name="reminder_notification_text">Remember to report any symptoms you might be experiencing.</string>
+    <string name="reminder_notification_text">Remember to report any symptoms you might be experiencing. Have you checked if you can smell today?</string>
```

Comment:
"Remember to report any symptoms you might be experiencing, including loss of smell."